webui/react/src/components/InlineEditor.module.scss

```diff
@@ -51,10 +51,18 @@
     border-color: var(--theme-colors-monochrome-14);
   }
   .growWrap > .spinner {
-    display: hidden;
+    display: none;
     position: absolute;
-    right: 18px;
-    top: 18px;
+    top: 45%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+  .growWrap > .backdrop {
+    display: none;
+    position: absolute;
+    background-color: rgba(0, 0, 0, .4);
+    width: 100%;
+    height: 100%;
```

Comment:
added a backdrop to improve the visibility and "hierarchy" of the spinner in the component.

https://user-images.githubusercontent.com/104855841/174311746-1752801c-4998-4bee-8db1-bf79d9546f0e.mp4